libs-haskell/test-utils/DA/Test/Calling.hs

```diff
@@ -0,0 +1,69 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module DA.Test.Calling
```

Comment:
`DA.Test.Calling` sounds very confusing. Given that the stuff already moved to `DA.Test.Util` I would leave it in there. If you really want to use a separate module use `DA.Test.Process`.

Reply:
Renamed to DA.Test.Process